flask-api/app/routes/routes.py

```diff
@@ -1,8 +1,9 @@
 from app import app
-from ..views import users
+from ..views import users, helper
 from flask import jsonify
 
 @app.route('/', methods=['GET'])
+@helper.token_required
 def root():
```

Comment:
add `current_user` to root function, since 'helper.token_required' call him with this argument.